src/client/utils/local-storage/local-storage.ts

```diff
@@ -0,0 +1,41 @@
+var isSupported = false;
+
+var mod = 'test';
+try {
+    localStorage.setItem(mod, mod);
+    localStorage.removeItem(mod);
+    isSupported = true;
+} catch (e) {
+    isSupported = false;
+}
+
+export function supported(): boolean {
+  return isSupported;
+}
+
+export function get(key: string): any {
+  if (!isSupported) {
+    throw new Error('LocalStorage not supported');
+  }
```

Comment:
It is crazy for get to throw an error if localstorage is not supported. Instead you should pass a second argument for the default value and that should be (quietly) used.